### PR TITLE
A11y: Fix aria attribute for page number and add correct role

### PR DESCRIPTION
**Pull request by amreladawy**
Run FastPass on Paginating control. 
It will report an error that aria-selected is a wrong attribute for page number button. The correct attribute is aria-checked. Also, added role radio to reflect the correct role of this pagination.
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-selected
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-checked

## Previous Behavior

The pagination buttons have the wrong aria attribute and fail a11y test tool by Microsoft.

## New Behavior

The buttons have the correct aria-checked and correct role

## Related Issue(s)

- Fixes #

### CODE REVIEW

File: packages/react-experiments/src/components/Pagination/PageNumber.tsx [hunk 1]
```diff
@@ -14,9 +14,10 @@ export class PageNumber extends React.Component<IPageNumberProps, {}> {
       <DefaultButton
         key={page}
         onClick={this._onClick}
-        aria-selected={selected}
+        aria-checked={selected}
         aria-label={ariaLabel}
         styles={{ root: className }}
+        role="radio"
```
Comment (behowell) [inline packages/react-experiments/src/components/Pagination/PageNumber.tsx:20]: The [MDN documentation for the radio role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/radio_role) says they must be grouped inside an element with role="radiogroup". I couldn't immediately verify that's the case--do you need to add that to the parent element?

**Review by behowell — APPROVED** [review]

File: packages/react-experiments/src/components/Pagination/PageNumber.tsx (hunk 1 above)
Comment (amreladawy) [reply packages/react-experiments/src/components/Pagination/PageNumber.tsx:20]: Sure, will raise a separate PR for that!

**Comment by amreladawy** [comment]
Updated the parent control and added  role="radiogroup"

**Comment by amreladawy** [comment]
Closed and raised another PR https://github.com/microsoft/fluentui/pull/29349

**Pull request closed without merging**